website/allowed-subcategories.txt

```diff
@@ -63,6 +63,7 @@ Kinesis
 Kinesis Data Analytics v2 (SQL and Java Applications)
 Kinesis Firehose
 Kinesis Video
+LakeFormation
```

Comment:
Nit. `Lake Formation` as that's the [official spelling](https://aws.amazon.com/lake-formation/) of the service.

Reply:
Updated using official spelling.